devdox/app/utils/repo_fetcher.py

```
@@ -1,36 +1,40 @@
-from typing import Callable, List, Protocol, Tuple
+from typing import Any, Callable, List, Protocol, Tuple
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import

(F401)